tests/scale/test_load.py

```diff
@@ -477,9 +477,9 @@ def _create_service_accounts(service_name, security=None):
                 sa_name, sa_secret, sanitized_service_name
             )
 
-            sdk_security.grant_permissions("root", "*", sa_name)
+            sdk_security.grant_permissions("nobody", "*", sa_name)
 
-            sdk_security.grant_permissions("root", SHARED_ROLE, sa_name)
+            sdk_security.grant_permissions("nobody", SHARED_ROLE, sa_name)
```

Comment:
I'm not sure if this is enough nor should this be hard coded. It seems the permissions have a different concept than the usual permission paths.